testsuite/model/src/main/java/org/keycloak/testsuite/model/HotRodServerRule.java

```diff
@@ -104,6 +105,9 @@ private void createKeycloakCaches(boolean async, String... cache) {
                 .site("site-1").backupFailurePolicy(BackupFailurePolicy.IGNORE).strategy(BackupConfiguration.BackupStrategy.SYNC)
```

Comment:
@graziang Set the backup failure policy `BackupFailurePolicy.FAIL` here and in line 102. It ignores the deadlock and assumes the update is applied.